src/content/api/payment-requests.mdx

```diff
@@ -576,7 +577,7 @@ Payment Activities are created when a Payment Request has been `created`, `paid`
   path="/api/payment-requests/{paymentRequestId}/summary"
   filename="payment-requests-get-summary"
 >
-  ## Get Payment Request Summary
+  ## Get Payment Request Summary <Badge text="public" />
```

Comment:
Was the design to use the default color?

Reply:
Yea it was but @tom-blake didn't like adding the public badge as all the endpoints are public. There is a discussion in the design channel about alternative designs